pkg/monitor/nftables_linux.go

```
@@ -466,3 +469,213 @@ func checkHAProxyFirewallRules(apiVip string, apiPort, lbPort uint16) (bool, err
+	}
+
+	if err := conn.Flush(); err != nil {
+		return fmt.Errorf("failed to flush nftables changes: %v", err)
+	}
+
+	return nil
+}
+
+// checkCorednsFirewallRules reports whether both CoreDNS block rules (TCP and
+// UDP) exist.
+func checkCorednsFirewallRules(port uint16) (bool, error) {
```

golangci-lint (2.13.2)

[error] func checkCorednsFirewallRules is unused

(unused)